apis/externalsecrets/v1beta1/secretstore_vault_types.go

```diff
@@ -187,6 +187,7 @@ type VaultKubernetesServiceAccountTokenAuth struct {
 	// Optional audiences field that will be used to request a temporary Kubernetes service
 	// account token for the service account referenced by `serviceAccountRef`.
 	// Defaults to a single audience `vault` it not specified.
+	// Deprecated: use serviceAccountRef.Audiences instead
```

Comment:
I want to deprecate this in favor of `serviceAccountRef.Audiences[]`.

It looks like `expirationSeconds` is not needed because we generate a service account token on the fly and use it only once.  There is no replacement for `expirationSeconds`